sceptre/cli/list.py

```diff
@@ -156,11 +156,11 @@ def list_stacks(ctx, path):
     write(output, output_format)
 
 
-@list_group.command(name="sceptre_user_data")
+@list_group.command(name="config")
 @click.argument("path")
 @click.pass_context
 @catch_exceptions
-def list_stacks(ctx, path):
+def list_config(ctx, path):
```

Comment:
It doesn't really seem like a `list` command, though. Maybe `render` would be more accurate.